src/synology_dsm/api/core/utilization.py

```diff
@@ -1,164 +1,197 @@
 """DSM Utilization data."""
+from __future__ import annotations
+
+from typing import TypedDict
+
 from synology_dsm.api import SynoBaseApi
 from synology_dsm.helpers import SynoFormatHelper
 
-
-class SynoCoreUtilization(SynoBaseApi):
+CpuUtilization = TypedDict(
+    "CpuUtilization",
+    {
+        "15min_load": int,
+        "1min_load": int,
+        "5min_load": int,
+        "device": str,
+        "other_load": int,
+        "system_load": int,
+        "user_load": int,
+    },
+)
+
+MemoryUtilization = TypedDict(
+    "MemoryUtilization",
+    {
+        "avail_real": int,
+        "avail_swap": int,
+        "buffer": int,
+        "cached": int,
+        "device": str,
+        "memory_size": int,
+        "real_usage": int,
+        "si_disk": int,
+        "so_disk": int,
+        "swap_usage": int,
+        "total_real": int,
+        "total_swap": int,
+    },
+)
+
+NetworkUtilization = TypedDict(
+    "NetworkUtilization",
+    {
+        "device": str,
+        "rx": int,
+        "tx": int,
+    },
+)
+
+
+class DataType(TypedDict, total=False):

```

Comment:
I'd probably call this something more specific than `Data` (naming is hard)

Reply:
good point. many thanks for your review 🙂